docs/examples/irradiance-transposition/plot_rtranpose_year.py

```diff
@@ -0,0 +1,137 @@
+"""
+Reverse transposition using one year of hourly data
```

Comment:
I guess this title is misleading now that you're not actually simulating "one year of hourly data".
```suggestion
Reverse transposition
```